ocaml/xapi/xapi_globs.ml

```diff
@@ -988,6 +988,9 @@ let max_spans = ref 1000
 
 let max_traces = ref 10000
 
+(** 1MB *)
+let max_observer_file_size = ref 1048576
```

Comment:
Would be better to express these numbers as a product of powers of two - otherwise it's not clear why we have seemingly special numbers. Or write a functions like `mib` and `kib` to denote common sizes.